[2021-11-01 02:23:03,799] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.create_table_passengers manual__2021-11-01T02:23:02.969448+00:00 [queued]>
[2021-11-01 02:23:03,813] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: my_dag.create_table_passengers manual__2021-11-01T02:23:02.969448+00:00 [queued]>
[2021-11-01 02:23:03,813] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 02:23:03,813] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-01 02:23:03,813] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-01 02:23:03,823] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): create_table_passengers> on 2021-11-01 02:23:02.969448+00:00
[2021-11-01 02:23:03,827] {standard_task_runner.py:52} INFO - Started process 2423 to run task
[2021-11-01 02:23:03,830] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'create_table_passengers', 'manual__2021-11-01T02:23:02.969448+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/checkModel.py', '--cfg-path', '/tmp/tmp88t_os1p', '--error-file', '/tmp/tmphhnsb04m']
[2021-11-01 02:23:03,830] {standard_task_runner.py:77} INFO - Job 89: Subtask create_table_passengers
[2021-11-01 02:23:03,860] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.create_table_passengers manual__2021-11-01T02:23:02.969448+00:00 [running]> on host 93c37aa78767
[2021-11-01 02:23:03,897] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=create_table_passengers
AIRFLOW_CTX_EXECUTION_DATE=2021-11-01T02:23:02.969448+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-01T02:23:02.969448+00:00
[2021-11-01 02:23:03,904] {base.py:79} INFO - Using connection to: id: postgres_default. Host: 172.23.0.3, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2021-11-01 02:23:03,906] {dbapi.py:225} INFO - Running statement: CREATE TABLE if not exists passengers (passengerid int primary key, name text, sex text, age float, sibsp int, parch int, ticket text, fare float, cabin text, embarked text);, parameters: None
[2021-11-01 02:23:03,907] {postgres.py:71} INFO - NOTICE:  relation "passengers" already exists, skipping

[2021-11-01 02:23:03,916] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=create_table_passengers, execution_date=20211101T022302, start_date=20211101T022303, end_date=20211101T022303
[2021-11-01 02:23:03,961] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-01 02:23:04,011] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
